apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.UnitTests/PriorityListTests.cs

```diff
@@ -0,0 +1,235 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Google.Cloud.Spanner.V1;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.Tests
+{
+    public class PriorityListTests
+    {
+        [Fact]
+        public void CanAddMultipleOfSamePriority()
+        {
+            PriorityList<TestItem> priorityList = new PriorityList<TestItem>();
+            TestItem item1, item2;
+            priorityList.Add(item1 = new TestItem(5));
+            priorityList.Add(item2 = new TestItem(5));
+            Assert.Equal(2, priorityList.Count);
+            AssertExist(priorityList, item1, item2);
```

Comment:
Not really.  if the priority is the same, then the relative position is not deterministic.